schemas/2015-01-01/deploymentTemplate.json

```diff
@@ -531,7 +531,7 @@
                   { "$ref": "https://schema.management.azure.com/schemas/2017-10-01/Microsoft.Network.json#/resourceDefinitions/dnsZones_PTR" },
                   { "$ref": "https://schema.management.azure.com/schemas/2017-10-01/Microsoft.Network.json#/resourceDefinitions/dnsZones_SOA" },
                   { "$ref": "https://schema.management.azure.com/schemas/2017-10-01/Microsoft.Network.json#/resourceDefinitions/dnsZones_SRV" },
-                  { "$ref": "https://schema.management.azure.com/schemas/2017-10-01/Microsoft.Network.json#/resourceDefinitions/dnsZones_TXT" },
+                  { "$ref": "https://schema.management.azure.com/schemas/2017-10-01/Microsoft.Network.json#/resourceDefinitions/dnsZones_TXT" },				  
```

Comment:
>				   [](start = 144, length = 6)

please remove the extra tabs and spaces.

Reply:
removed. Thanks!